core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -357,6 +361,26 @@ static Schema copyRecord(Schema record, List<Schema.Field> newFields, String new
     return copy;
   }
 
+  static Schema copyArray(Schema array, Schema elementSchema) {
```

Comment:
I don't think that `copy` is quite the correct verb here because this is not actually copying the array. It is replacing the element. What about `replaceElement` and `replaceValue`?

Reply:
Adopted `replaceElement` and `replaceValue`